es-design-system/aws/package.json

```diff
@@ -11,7 +11,7 @@
   "dependencies": {
     "aws-cdk": "^2.69.0",
     "aws-cdk-lib": "^2.69.0",
-    "es-cdk": "^1.20.0",
```

Comment:
I'd keep this on the same version as `main` for now if possible.

Reply:
It is.